BR FAQ: add a warning about multi br importing (#6263)
pingcap/docs: br/backup-and-restore-faq.md

In v4.0.9, BR backs up statistics by default, which consumes too much memory. To ensure that the backup process goes well, the backup for statistics is disabled by default starting from v4.0.10.

If you do not execute `ANALYZE` on the table, TiDB will fail to select the optimized execution plan due to inaccurate statistics. If query performance is not a key concern, you can ignore `ANALYZE`.

> ## Can I use multiple BR command-lines at a single time to restore the data of a single cluster?

Reviewer: @YuJuncen Can we use "BR commands"? The Chinese version is "BR 命令". Personally, I think "BR processes" make more sense. Or can you help create a docs-cn PR to modify the Chinese to "BR 进程"？

> ## Can I use multiple BR command-lines at a single time to restore the data of a single cluster?

Reviewer: @en-jin19 "at a single time" is really confusing.

> ## Can I use multiple BR command-lines at a single time to restore the data of a single cluster?

Reviewer 2: > @YuJuncen Can we use "BR commands"? The Chinese version is "BR 命令". Personally, I think "BR processes" make more sense. Or can you help create a docs-cn PR to modify the Chinese to "BR 进程"？

Sure

> ## Can I use multiple BR command-lines at a single time to restore the data of a single cluster?

Author: ```suggestion
## Can I use multiple BR command-lines at the same time to restore the data of a single cluster?
```

> ## Can I use multiple BR command-lines at a single time to restore the data of a single cluster?

Author: > @en-jin19 "at a single time" is really confusing. Please choose a proper expression.

how about using "at the same time" instead of it？

> ## Can I use multiple BR command-lines at the same time to restore the data of a single cluster?

Reviewer: ```suggestion
## Can I use multiple BR processes at the same time to restore the data of a single cluster?
```

> **It is strongly not recommended** to use multiple BR command-lines at a single time to restore the data of a single cluster for the following reasons:

Author: ```suggestion
**It is strongly not recommended** to use multiple BR command-lines at the same time to restore the data of a single cluster for the following reasons:
```

> **It is strongly not recommended** to use multiple BR command-lines at the same time to restore the data of a single cluster for the following reasons:

Reviewer: ```suggestion
**It is strongly not recommended** to use multiple BR processes at the same time to restore the data of a single cluster for the following reasons:
```

> + When BR restores data, it modifies some of the global configurations of PD. Therefore, if you use multiple BR processes for data restore at the same time, these configurations might be mistakenly overwritten and cause abnormal cluster status.

Reviewer: ```suggestion
+ When BR restores data, it modifies some global configurations of PD. Therefore, if you use multiple BR processes for data restore at the same time, these configurations might be mistakenly overwritten and cause abnormal cluster status.
```

> + When BR restores data, it modifies some of the global configurations of PD. Therefore, if you use multiple BR command-lines for data restoration at a single time, these configurations might be mistakenly overridden and cause abnormal cluster status.

Author: ```suggestion
+ When BR restores data, it modifies some of the global configurations of PD. Therefore, if you use multiple BR command-lines for data restoration at the same time, these configurations might be mistakenly overridden and cause abnormal cluster status.
```

> + When BR restores data, it modifies some of the global configurations of PD. Therefore, if you use multiple BR command-lines for data restoration at the same time, these configurations might be mistakenly overridden and cause abnormal cluster status.

Reviewer: ```suggestion
+ When BR restores data, it modifies some of the global configurations of PD. Therefore, if you use multiple BR processes for data restore at the same time, these configurations might be mistakenly overwritten and cause abnormal cluster status.
```

> + BR uses a lot of cluster resources to restore data, so in fact, running BR command-lines in parallel has very limited effects to improve the speed of restoration.

Reviewer: ```suggestion
+ BR consumes a lot of cluster resources to restore data, so in fact, running BR processes in parallel improves the restore speed only to a limited extent.
```

> + There was no test for running multiple BR command-lines in parallel for data restoration, so it is not guaranteed to succeed.

Reviewer: ```suggestion
+ There has been no test for running multiple BR processes in parallel for data restore, so it is not guaranteed to succeed.
```
